python/tidy/servo_tidy/tidy.py

```diff
@@ -1052,16 +1052,24 @@ def run_lint_scripts(only_changed_files=False, progress=True, stylo=False):
 
 
 def check_commits(path='.'):
+    """Checks if the test is being run under Travis CI environment"""
+    is_travis = (os.environ.get('TRAVIS') == 'true')
+    number_commits = '-n2' if is_travis else '-n1'
+
     """Gets all commits since the last merge."""
-    args = ['git', 'log', '-n1', '--merges', '--format=%H']
-    last_merge = subprocess.check_output(args, cwd=path).strip()
-    args = ['git', 'log', '{}..HEAD'.format(last_merge), '--format=%s']
+    args = ['git', 'log', number_commits, '--merges', '--format=%H %an']
+    # last_merge stores both the commit hash and the author name of the last merge in the output
+    last_merge_hash, last_merge_author = subprocess.check_output(args, cwd=path).strip().split('\n')[-1].split(None, 1)
```

Comment:
Oh, right. Maybe we could ask `git` for a helpful format, like `--format="%H:%an"` and then do,

``` python
subprocess.check_output(...).rstrip().split('\n')[-1].split(':')
```

I still believe that `split(None, 1)` would be confusing.